rgw/v2/lib/resource_op.py

```diff
@@ -208,6 +208,9 @@ def read(self):
         self.version_enable = self.doc["config"].get("version_enable", False)
         self.copy_version_object = self.doc["config"].get("copy_version_object", False)
         self.object_expire = self.doc["config"].get("object_expire", False)
+        self.multisite_dynamic_resharding = self.doc["config"].get(
```

Comment:
I would suggest using a different variable name rather than `multisite_dynamic_resharding`. just `dynamic_resharding` sounds good I believe. what do you think?